backend/src/main/java/com/redhat/cloud/notifications/auth/rbac/RbacIdentityProvider.java

```diff
@@ -30,6 +30,7 @@
 @ApplicationScoped
 public class RbacIdentityProvider implements IdentityProvider<RhIdentityAuthenticationRequest> {
 
+    public static final String RBAC_READ_EVENTS_NOTIFICATIONS = "read:events:notifications";
```

Comment:
This name is more concrete and mimics exactly what we should have in RBAC, but is not consistent with our other internal RBAC permissions. RBAC server permissions have a `verb`, a `resource` and an `application` while our internal ones only have a `verb` and a `resource`. 

No strong opinion, but to be consistent we should rename it to have the `verb` and `resource`.

```suggestion
    public static final String RBAC_READ_EVENTS_NOTIFICATIONS = "read:events";
```

Reply:
Changed